src/hypothesis/serializers/hypothesis_serializer.py

```diff
@@ -59,6 +60,7 @@ class Meta:
             'slug',
             'src',
             'unified_document',
+            'unified_document_id',
```

Comment:
When creating reviews on  the FE, posts and papers have   `unified_document_id` but hypothesis does not. This leads to inconsistency.

Reply:
I agree with the consistency issue. I'm not sure why but the posts and papers serializer includes `unified_document_id`, when `unified_document` works. I think we should get rid of `unified_document_id` and and change the other serializers to use `unified_document` instead.